.github/workflows/php-tests.yml

```diff
@@ -45,7 +44,7 @@ jobs:
           chmod 000 server_certificates_bundle_unreadable.pem
 
       - name: Run PHPUnit
-        run: phpunit -c phpunit.xml
+        run: ./vendor/bin/phpunit -c phpunit.xml
```

Comment:
I'm not convinced this is a change I want. Either way, this should be in a separate PR